resources/buffs.js

```diff
@@ -151,8 +151,9 @@ class BuffsUI extends HTMLElement {
   onPlayerChanged(e) {
     this._me = e.detail.name;
     this._buff_regex = new RegExp(':' + this.me + ' gains the effect of (.+) from (.*) for ([0-9.]+) Seconds\.$');
-    document.removeEventListener('onPlayerChangedEvent', this.onPlayerChanged);
-    document.addEventListener('onLogEvent', this.onLogEvent);
+    
+    removeOverlayListener('onPlayerChangedEvent', this.onPlayerChanged);
```

Comment:
Hmm, this file isn't used at all.  I think this is some sort of vestigial code that preceded the jobs buff tracker.  I'm going to delete it.